src/components/NavItems.tsx

```diff
@@ -0,0 +1,52 @@
+import { Typography } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import { NavPages } from "../constants/NavPages";
+import { buildPath } from "../utils/pathBuilder";
+
+type NavItemsProps = {
+  activePath: string;
+  variant: "mobile" | "desktop";
+};
+
+const NavItems = ({ activePath, variant }: NavItemsProps) => {
+  const navigate = useNavigate();
+
+  const desktopStyles = {
+    margin: 1,
+  };
+
+  const mobileStyles = {
```

Comment:
these, I would write outside the NavItems. before 11th line. 

  const DESKTOP_STYLES = {
    margin: 1,
  };


constants naming usually written uppercased.